reload.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"github.com/Qv2ray/mmp-go/config"
+	"github.com/Qv2ray/mmp-go/dispatcher"
+	"log"
+	"sync"
+)
+
+var mMutex sync.Mutex
+var mPortDispatcher = make(map[int]*[len(protocols)]dispatcher.Dispatcher)
+
+func ReloadConfig() {
+	log.Println("configure reloading")
+	mMutex.Lock()
+	defer mMutex.Unlock()
+
+	// rebuild config
+	confPath := config.GetConfig().ConfPath
+	newConf, err := config.BuildConfig(confPath)
+	if err != nil {
+		log.Printf("failed to reload: %v", err)
```

Comment:
```suggestion
		log.Printf("failed to reload configuration: %v", err)
```

also consider abort?